views/index.haml

```diff
@@ -1,17 +1,16 @@
 %body
 
-%form{:action => "#"}
-  %h1 Todos
-  %fieldset
```

Comment:
I missed that one. I was really confused why the browser behaved awkwardly. It was sending GET requests even though I specified POST in the form. This `fieldset` was the reason why.